mayhem/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Copyright 2022 Google LLC.
```

Comment:
Copyright and package stage.

Reply:
I believe the copyright is necessary here.